docs/userguide/configuration.rst

```diff
@@ -150,6 +150,7 @@ have been moved into a new  ``task_`` prefix.
 ``CELERY_TASK_TRACK_STARTED``              :setting:`task_track_started`
 ``CELERY_TASK_REJECT_ON_WORKER_LOST``      :setting:`task_reject_on_worker_lost`
 ``CELERYD_TIME_LIMIT``                     :setting:`task_time_limit`
+``CELERY_TASK_RESULTREPR_MAXSIZE``         :setting:`task_resultrepr_maxsize` 
```

Comment:
i don't think we need `CELERY_TASK_RESULTREPR_MAXSIZE`` as there shouldn't be an equivalant in upper case